pkgs/applications/audio/cmus/default.nix

```
@@ -92,31 +92,15 @@ in
 
 stdenv.mkDerivation rec {
   pname = "cmus";
-  version = "2.10.0";
+  version = "master-2023-11-05";
```

```suggestion
  version = "2.10.0-unstable-2023-11-05";
```
see the contributing guide on the version attr